src/cli/commands/codegen/route-types.ts

```diff
@@ -103,7 +115,11 @@ export class CodeGenRouteTypes extends BaseCommand {
     const renderType = <TType extends ts.Type>(type: Type<TType>) => {
       return project
         .getTypeChecker()
-        .compilerObject.typeToString(type.compilerType)
+        .compilerObject.typeToString(
+          type.compilerType,
+          undefined,
+          TypeFormatFlags.NoTruncation
```

Comment:
I'm assuming this is for convenience rather than functionality, right? So types aren't super obscure?

In any case can you introduce a test case for this?

Reply:
without this deeply nested types were being printed with `...` (similar to what you see when hovering in VS Code) resulting in invalid code

can try to reproduce in test 👍

Reply:
hmmmmm i hope there aren't more cases than this that would cause this 🤔 i guess we'll find out

Reply:
yeah me too haha
fake-2n has some decently narly types so I think we should be good

Reply:
questioning my sanity because I can't reproduce it now in fake-2n

will remove for now